lib/Elastica/Client.php

```diff
@@ -640,6 +647,14 @@ public function request($path, $method = Request::GET, $data = array(), array $q
 
             return $this->request($path, $method, $data, $query);
         }
+
+        $this->_logger->debug('Elastica Request', [
```

Comment:
If a document had a reference to something that couldn't be serialized it could become a problem - except I suspect Elastica itself wouldn't work in that circumstance anyway.